builder/BuilderSatFix.sol

```diff
@@ -0,0 +1,49 @@
+//
+// AIRA Builder for AirTrafficController contract
+//
+//
+
+import 'creator/CreatorSatFix.sol';
+
+/**
+ * @title BuilderSatFix contract
+ */
+contract BuilderSatFix {
+
+    /* The last builded SatFix */
+    address public getLastContract = 0x0;
+
+    /**
+     * @dev this event emitted for every builded contract
+     */
+    event Builded(address indexed sender, address indexed instance);
+
+    /**
+     * @dev Run script creation contract
+     * @param _latitude is an latitude
+     * @param _longitude is an longitude
+     * @param _altitude is an altitude
+     * @return address new contract
+     */
+    function create(int256 _latitude,
+                    int256 _longitude,
+                    int256 _altitude)
+                    returns (address) {
+        var inst = CreatorSatFix.create(_latitude, _longitude, _altitude);
+        getLastContract = inst;
```

Comment:
Is not a good way for multiuser configuration. Use `mapping(address=>address)` for sender driven last contract access or the best is `mapping(address=>address[])` for store all contract of user.